### PR TITLE
Remove stale users

### PR DESCRIPTION
**Pull request by g4s8**
#835 - added `Delete user` stakeholder, also bug submitted: https://github.com/jcabi/jcabi-github/issues/1370

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#851](https://github.com/zerocracy/farm/issues/851) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#851](https://github.com/zerocracy/farm/issues/851) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/791340592); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

File: src/main/resources/com/zerocracy/stk/pmo/people/delete_user.groovy [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import com.zerocracy.pm.ClaimIn
+import com.zerocracy.pmo.People
+
+def exec(Project pmo, XML xml) {
+  new Assume(pmo, xml).isPmo()
+  new Assume(pmo, xml).type('Delete user')
+  ClaimIn claim = new ClaimIn(xml)
+  String user = claim.param('login')
+  People people = new People(pmo).bootstrap()
+  people.remove(user)
```
Comment (carlosmiranda) [inline src/main/resources/com/zerocracy/stk/pmo/people/delete_user.groovy]: @g4s8 we're only using this variable once, let's inline call to `remove`.

**Comment by g4s8** [comment]
@carlosmiranda fixed ca7178d please look again

**Comment by g4s8** [comment]
@yegor256 ping

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 Rultor failed to push here, please restart merge

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 10days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 10days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by g4s8** [comment]
@yegor256 ping

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#851](https://github.com/zerocracy/farm/issues/851) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda Thanks!

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Pull request merged**